basics/src/main/kotlin/de/thermondo/basics/Basics06.kt

```diff
@@ -0,0 +1,12 @@
+package de.thermondo.basics
+
+/*
+ * You can pass parameters to classes via the class constructor by first giving the parameter name
+ * and then the type, similar to how we create functions with parameters.
+ *
+ * Let's create a class called Basics06, that accepts the following parameters:
+ * @param name: String
+ * @param age: Int
+ * and return the name and age of the two people.
```

Comment:
```suggestion
 *
 * Define member variables:
 * userName: String
 * userAge: Int
 *
 * and assign these member variables the parameter values.
```
This is a better description of what we want to see implemented. This also comes with a few smaller changes.